deploy/crds/push_v1alpha1_unifiedpushserver_crd.yaml

```diff
@@ -69,6 +69,30 @@ spec:
                 - backendSecretName
                 type: object
               type: array
+            database:
+              properties:
+                databaseHost:
+                  description: Database host for external database support
+                  type: string
+                databaseName:
+                  description: Database name for external database support
+                  type: string
+                databasePassword:
+                  description: Database password for external database support
+                  type: string
+                databasePort:
```

Comment:
This value is not used in the secret [1]. Also it is hardcoded in container environment [2].

[1] https://github.com/aerogear/unifiedpush-operator/pull/86/files#diff-a02d8180a2f035b2a960322ad76cd7baR55
[2] https://github.com/aerogear/unifiedpush-operator/blob/8ab2aff005ce988240917ac7e2bd08655f6bd050/pkg/controller/unifiedpushserver/unifiedpushserver.go#L86-L89

Reply:
good catch :+1: